tests/display/src/RpcDisplayTestCase.hx

```diff
@@ -15,7 +15,7 @@ class RpcDisplayTestCase implements utest.ITest {
 
 	@:timeout(3000)
 	public function setup() {
-		var methodArgs = {method: Methods.ResetCache, id: 1, params: {}};
+		static var methodArgs = {method: Methods.ResetCache, id: 1, params: {}};
 		var args = ['--display', Json.stringify(methodArgs)];
```

Comment:
Well, this one too if you need another test run!

Reply:
Yep, was keeping it just for that :grin:

Reply:
Well...

```
 ERROR  src/XmlDisplayTestCase.hx:16: characters 35-66

 16 |   static var args = ['--display', haxe.Json.stringify(methodArgs)];
    |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    | Accessing local variables in static initialization is not allowed
```

Reply:
Huh, it checks `has_var_flag v VStatic`...